plugins/record-hook/src/index.ts

```diff
@@ -0,0 +1,23 @@
+import { Client, FlatfileEvent } from '@flatfile/listener'
+import { RecordHook } from '@flatfile/configure'
+import { FlatfileRecord } from '@flatfile/hooks'
+
+export const recordHook = (
+  sheetSlug: string,
+  callback: (record: FlatfileRecord) => {}
+) => {
+  return (client: Client) => {
+    client.on(
+      'records:*',
+      {
+        context: {
+          // @ts-ignore
```

Comment:
:suspect:

Reply:
@mairechew let me know that the and @bangarang are working on a fix for this